homeassistant/components/ted6000/manifest.json

```diff
@@ -0,0 +1,9 @@
+{
+  "domain": "ted6000",
```

Comment:
We should not add a new domain for each model.
We should have `ted` as the domain that works with both models.